dispatch/static/manager/webpack.config.js

```diff
@@ -50,5 +50,11 @@ module.exports = {
     }),
     new webpack.optimize.UglifyJsPlugin(),
     new ExtractTextPlugin('manager-' + version + '.css')
-  ]
+  ],
+  node: {
+    console: true,
+    fs: 'empty',
+    net: 'empty',
+    tls: 'empty'
+  }
```

Comment:
What's this for?

Reply:
I faced a problem in which the console would show "Error: Cannot find module "net"", and I solved it by adding that as explained here: https://github.com/request/request/issues/1529